src/test/java/io/confluent/idesidecar/restapi/resources/TemplateResourceIT.java

```diff
@@ -1,118 +0,0 @@
-package io.confluent.idesidecar.restapi.resources;
-
-import static io.confluent.idesidecar.restapi.util.ResourceIOUtil.asJson;
-import static io.restassured.RestAssured.given;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.wildfly.common.Assert.assertTrue;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import io.confluent.idesidecar.restapi.models.ApplyTemplateRequest;
-import io.confluent.idesidecar.scaffolding.models.TemplateManifest;
-import io.confluent.idesidecar.restapi.testutil.NoAccessFilterProfile;
-import io.quarkus.test.junit.QuarkusIntegrationTest;
-import io.quarkus.test.junit.TestProfile;
-import java.security.SecureRandom;
-import java.util.HashMap;
-import java.util.Optional;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
-
-@QuarkusIntegrationTest
-@Tag("io.confluent.common.utils.IntegrationTest")
-@TestProfile(NoAccessFilterProfile.class)
-public class TemplateResourceIT {
-
-  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-  private static final SecureRandom SECURE_RANDOM = new SecureRandom();
-  private static final int RANDOM_STRING_LENGTH = 25;
-
-  @Test
-  void shouldBeAbleToApplyAllTemplates() throws JsonProcessingException {
-    // Get all templates
-    var responseBody = given()
-        .when()
-        .get("/gateway/v1/templates")
```

Comment:
this is the same url now yeah? would we want to test this functionality still at the proxy level?

Reply:
Hm, I don't think we _can_ test the functionality at proxy level using integration tests unless we hit stag/devel CCloud service and I'm not sure we want to do that. Testing that payloads are sent and received properly from the service should be sufficient and this is tested in `TemplateResourceTest`.